packages/common/src/tid.ts

```diff
@@ -32,6 +32,10 @@ export class TID {
     return TID.fromTime(timestamp, clockid)
   }
 
+  static nextStr(): string {
```

Comment:
tempted to make this whole thing not a class. not really giving us anything anymore